chain/near.ts

```diff
@@ -0,0 +1,386 @@
+import '../common/eager_offset'
+import { Bytes } from '../common/collections'
+import { BigInt } from '../common/numbers'
+
+// Most types from this namespace are direct mappings or adaptations from:
+// https://github.com/streamingfast/proto-near/blob/develop/sf/near/codec/v1/codec.proto
+export namespace near {
+  export type CryptoHash = Bytes
+
+  export type AccountId = string
+
+  export type BlockHeight = u64
+
+  export type Balance = BigInt
+
+  export type Gas = u64
+
+  export type ShardId = u64
+
+  export type NumBlocks = u64
+
+  export type ProtocolVersion = u32
+
+  export type Payload = u64
+
+  export enum CurveKind {
+    ED25519 = 0,
+    SECP256K1 = 1,
+  }
+
+  export class Signature {
+    constructor(
+      public kind: CurveKind,
+      public bytes: Bytes,
+    ) {}
+  }
+
+  export class PublicKey {
+    constructor(
+      public kind: CurveKind,
+      public bytes: Bytes,
+    ) {}
+  }
+
+  export enum AccessKeyPermissionKind {
+    FUNCTION_CALL = 0,
+    FULL_ACCESS = 1,
+  }
+
+  export class FunctionCallPermission {
+    constructor(
+      public allowance: BigInt,
+      public receiverId: string,
+      public methodNames: Array<string>,
+    ) {}
+  }
+
+  export class FullAccessPermission {}
+
+  export class AccessKeyPermissionValue {
+    constructor(
+      public kind: AccessPermissionKind,
```

Comment:
`AccessPermissionKind` is not defined anywhere.
```suggestion
      public kind: AccessKeyPermissionKind,
```

?

Reply:
You're correct, just fixed the typo, thanks!